### DISCUSSION TITLE
Xiaomi MiMouse2 support?

### DISCUSSION THREAD

**Discussion by viciious** [darthcloud/BlueRetro#1043] [General]
Mouse Model no: BXSBMW02
BlueRetro system: Sega Genesis

The mouse is listed as Bluetooth 4.2 compatible, so _in theory_ it should work. Pairing kinda works, at least the blue LED on the BlueRetro dongle stops glowing after the pairing mode on the mouse is activated, however that doesn't translate to mouse input in the actual game.

Pairing a 8bitdo M30 controller works fine, so I know the dongle itself works fine.

Bluetooth packet log attached.

[mimouse2.zip](https://github.com/user-attachments/files/16325791/mimouse2.zip)

**Comment by darthcloud** [comment]
Did you configure BlueRetro in mouse mode?
https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-BLE-Web-Config-User-Manual#81---mouse-config-example
